.github/workflows/lint-test.yml

```
@@ -1,30 +1,31 @@
-name: Lint and Test
+name: Build, Lint, and Test
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
 
 jobs:
-  lint-test:
-    name: Lint and Test
+  build-lint-test:
+    name: Build, Lint, and Test
     runs-on: ubuntu-20.04
     strategy:
       matrix:
         node-version: [12.x, 14.x, 16.x]
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: yarn --frozen-lockfile
-    - run: yarn lint
-    - run: yarn test
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn --ignore-scripts --frozen-lockfile
```

Probably better to omit `--ignore-scripts` until after we add `@lavamoat/allow-scripts`. I can do that in a separate PR
```suggestion
      - run: yarn --frozen-lockfile
```

---

Done in: 6025b58